pkg/tracing/names.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tracing
+
+import "fmt"
+
+// BrokerIngressNameArgs are the arguments needed to generate the BrokerIngressName.
```

Comment:
I guess the reasoning for this struct is that it names the arguments so they can be passed in any order and are self documenting. I'd probably use positional parameters instead, but I'm fine with this too.